### PR TITLE
Feat/cron added to run suite bh20 4700

### PR DESCRIPTION
**Pull request by sqrtCarlos**
Version 0.1 of the cron job to run the tests suite of ghost inspector.

### CODE REVIEW

File: src/createDeleteSubsScenario.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    const config = {
+        method: 'get',
+        url: process.env.SUBSCRIPTION_SUITE,
+    }
+    try {
+        await deleteSubscriptions(1);
+        await deletePaymentMethods();
+        await createMpPaymentMethod();
+        axios(config);
+        console.log('Done!');
```
Comment (davidenriquez-zxv) [inline src/createDeleteSubsScenario.js:18]: remove logs

File: src/createDeleteSubsScenario.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        url: process.env.SUBSCRIPTION_SUITE,
+    }
+    try {
+        await deleteSubscriptions(1);
+        await deletePaymentMethods();
+        await createMpPaymentMethod();
+        axios(config);
+        console.log('Done!');    
+    } catch (error) {
+        console.log(error);
```
Comment (davidenriquez-zxv) [inline src/createDeleteSubsScenario.js:20]: you should receive a message in slack or somewhere if there is any error

**Review by davidenriquez-zxv — CHANGES_REQUESTED** [review]
2 changes

File: src/createDeleteSubsScenario.js (hunk 1 above)
Comment (sqrtCarlos) [reply src/createDeleteSubsScenario.js:18]: These logs will be removed in the next commits.

File: src/utils/slack.utils.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    );
+    try {
+        return { success: res.data.ok };
+    } catch (error) {
+        return { success: false };
+    }
+}
+
+const sendMessage = async (message) => {
+    channels = String(process.env.SLACK_CHANNELS).trim();
```
Comment (davidenriquez-zxv) [inline src/utils/slack.utils.js:22]: why you dont use const to declare the variable?

**Pull request merged**